src/js/chat/helpers.js

```diff
@@ -114,6 +114,16 @@ exports.getCheerConfig = function(piece) {
     return config;
 };
 
+exports.dismissPinnedCheer = function() {
+    if (!window.App) return;
+    try {
+        var service = window.App.__container__.lookup('service:bits-pinned-cheers');
+        if (service.currentPinnedCheer && service.currentPinnedCheer !== null) service.dismissLocalMessage();
+    } catch (dismissError) {
+        debug.log('Failed to dismiss cheer: ' + dismissError);
```

Comment:
might want to not convert the error to string so it's an error object we can get a stack out of in the console..

`: ', dismissError` instead